### ISSUE TITLE
[Bug]: Badge label wrong class

### ISSUE THREAD

**Issue by skrepkaq** [piitaya/lovelace-mushroom#1529]
### Requirements

- [X] I checked the troubleshooting section in the README to verify that I have the latest Mushroom version.
- [X] I did a search to see if there is a similar issue or if a pull request is open.
- [X] I have read the state of card mod support (in pin issues) and understand card mod support is not provided in this repository.

### Current Behavior

Since version 4.0.6, the badge label is rendered with the wrong class. This seems to have been introduced by #1527, which changed class names only in the CSS, and not in the DOM.
![image](https://github.com/user-attachments/assets/5d1e94d3-80ad-44ba-8792-5c9cd2886100)

### Context

YAML state
```yaml
type: custom:mushroom-template-badge
content: Content
icon: mdi:mushroom
color: red
label: Label
```

### Environment

```markdown
- Browser: Chrome
- HA Version: 2024.8.3
- Mushroom: 4.0.6
```

**Comment by shbatm** [comment]
Confirmed seeing the same thing:

![image](https://github.com/user-attachments/assets/e0cbe524-b966-464f-8c79-1bc0f63fc4c8)
(left template badge, right built-in)

**Comment by benchdoos** [comment]
![image](https://github.com/user-attachments/assets/a1fd7fac-d49d-4d6a-86e1-57ab9ad4b332)

Same thing unfortunately🙁

**Comment by s1mpleman** [comment]
Same for me, since last update

**Comment by ambiman** [comment]
Same here :-(

**Comment by dannyvdbosch** [comment]
![IMG_3578](https://github.com/user-attachments/assets/77318a60-d194-4c02-881b-bbc0ad715af7)Same thing for me…

**Comment by piitaya** [comment]
Fixed with [`4.0.7`](https://github.com/piitaya/lovelace-mushroom/releases/tag/v4.0.7). Thank you for reporting the issue.
